src/pages/App.tsx

```diff
@@ -89,99 +71,47 @@ export default function App(): ReactElement {
   return (
     <QueryClientProvider client={queryClient}>
       <ReactQueryDevtools initialIsOpen />
-      <Suspense fallback={null}>
```

Comment:
suspense is kil?

Reply:
I am using `Suspense` under the `TopMenu` to wrap all pages lazy imported.